doc/en/getting-started.rst

```diff
@@ -28,7 +28,7 @@ Install ``pytest``
 .. code-block:: bash
 
     $ pytest --version
-    This is pytest version 5.x.y, imported from $PYTHON_PREFIX/lib/python3.8/site-packages/pytest/__init__.py
+    pytest 5.4.1.dev624+g4f318f86b.d20200707
```

Comment:
we probably need to do something in regen for this

Reply:
Oh good catch, missed that one. I will take a look later.

Reply:
Fixed, thanks!

Now it shows the actual version. 👍